Makefile

```diff
@@ -214,19 +214,25 @@ local-deploy-namespaced: docker-build kind-load-image ## Deploy the dns operator
 ##@ Build
 
 .PHONY: build
+build: GIT_SHA=$(shell git rev-parse HEAD || echo "unknown") 
+build: DIRTY=$(shell hack/check-git-dirty.sh || echo "unknown")
 build: manifests generate fmt vet ## Build manager binary.
-	go build -o bin/manager cmd/main.go
+	go build -ldflags "-X main.gitSHA=${GIT_SHA} -X main.dirty=${DIRTY}" -o bin/manager cmd/main.go
 
 .PHONY: run
+run: GIT_SHA=$(shell git rev-parse HEAD || echo "unknown") 
+run: DIRTY=$(shell hack/check-git-dirty.sh || echo "unknown")
```

Comment:
nit: I see we are basing these on what we have done in other repos, so we can leave it. My personal preference would have been to make them more explicit in their naming though https://github.com/hashicorp/vault-secrets-operator/blob/dcfee8e513e198aafad467ae71ec5cdd39269282/scripts/ldflags-version.sh#L52-L53

Reply:
We could do that, but the issue is needing to re-do on all other operators for consistency. The second round of this implementation can address this ?